README.md

```diff
@@ -1,2 +1,51 @@
-# annorxiver
-Annotating Rxiv preprints in an automated fashion
+# Annorxiver
+
+## Citation
+- [Manuscript in progress](greenelab.github.io/annorxiver_manuscript)
+- doi and full citation coming soon!!
+
+## About
+This repository contains code for the annorxvier project.
+This project focuses on analyzing the linguistic style and content of bioRxiv preprints and understanding how these features change when preprints undergo the publication process.
+We are currently working on a manuscript for the work in this repository.
+Feel free to check it out! (link above).
```

Comment:
Not sure if you will keep this sentence in the future, but if you do, maybe just add the link here `(link above)` >> `(link [here](https:/greenelab.github.io/annorxiver_manuscript)`